default.vcl

```diff
@@ -79,21 +104,25 @@ sub vcl_recv {
     	    return (synth(429, "Too Many Requests"));
         }
     } elseif (req.url ~ "^\/content\/notifications-push.*$") {
-        set req.backend_hint = content_notifications_push_api;
-    } elseif (req.url ~ "^\/lists\/notifications-push.*$") {
-        set req.backend_hint = list_notifications_push;
+        set req.backend_hint = content_notifications_push;
         # Routing preset here as vulcan is unable to route on query strings
```

Comment:
Once being active, now vulcan is unlikely to erupt again. I remove the references